zone/client_packet.cpp

```diff
@@ -5845,6 +5868,33 @@ void Client::Handle_OP_DoGroupLeadershipAbility(const EQApplicationPacket *app)
 		break;
 	}
 
+	case RaidLeadershipAbility_MainAssist:
+	{
+		//This is not needed as it executes from opcode 0x2b33 which is sent
+		//with this opcode.
+		//if (GetTarget())
+		//{
+		//	Raid* r = GetRaid();
+		//	if (r)
+		//	{
+		//		r->DelegateAbility(GetTarget()->CastToClient()->GetName());
+		//	}
+		//}
```

Comment:
Delete commented code?

Reply:
I decided to leave this as a reference as to what there is no code in this handler.  The RoF2 client sends this with the 2b33 opcode.  Other clients may not, and this would be a good reference to the next dev.  Happy to remove it if you like.  Left it for now pending direction.